packages/router/src/bindings/contractReader/index.ts

```diff
@@ -62,13 +62,14 @@ export const bindContractReader = async () => {
           };
         });
 
-        logger.info("active transactions tracker", requestContext, methodContext, {
+        logger.info("Trackers overview", requestContext, methodContext, {
```

Comment:
I really like having the tracker in info tbh.

Reply:
its there in debug, but when you're trying to dig down on something it blows up the logs